src/content/contributing/community-membership/_index.en.md

```diff
@@ -0,0 +1,193 @@
+- Reviewed at least 20 substantial PRs to the codebase
+- Knowledgeable about the codebase
+- Sponsored by two committers or project owners
+  - With no objections from other committers or project owners
+- May either self-nominate or be nominated by a committer/owner
+
+### Responsibilities and privileges
+
+The following apply to the part of codebase for which one would be a committer
+in an CODEOWNERS file:
+
+- Responsible for project quality control via [code reviews]
+  - Focus on code quality and correctness, including testing and factoring
+  - **Until automation supports approvers vs reviewers:** Focus on holistic
+    acceptance of contribution such as dependencies with other features,
+    backwards / forwards compatibility, API and flag definitions, etc
+- Expected to be responsive to review requests as per [community expectations]
+- Assigned PRs to review related to project of expertise
+- Assigned test bugs related to project of expertise
+- Granted "read access" to submariner repo
+- May get a badge on PR and issue comments
+- Demonstrate sound technical judgement
+- Mentor contributors and reviewers
+
+## Project Owner
+
+Project owners are the technical authority for the Submariner project. They
+*MUST* have demonstrated both good judgement and responsibility towards the
+health the project. Project owners *MUST* set technical direction and make or
+approve design decisions for the project - either directly or through
+delegation of these responsibilities.
+
+**Defined by:** Member of the [submariner-owners GitHub team][owners team].
+
+### Requirements
+
+Unlike the roles outlined above, the owners of the project are typically
+limited to a relatively small group of decision makers and updated as fits the
+needs of the project.
+
+The following apply to people who would be an owner:
+
+- Deep understanding of the technical goals and direction of the project
+- Deep understanding of the technical domain of the project
+- Sustained contributions to design and direction by doing all of:
+  - Authoring and reviewing proposals
+  - Initiating, contributing and resolving discussions (emails, GitHub issues,
+    meetings)
+  - Identifying subtle or complex issues in designs and implementation PRs
+- Directly contributed to the project through implementation and / or review
+
+### Responsibilities and privileges
+
+The following apply to people who would be an owner:
+
+- Make and approve technical design decisions for the project
+- Set technical direction and priorities for the project
+- Define milestones and releases
+- Mentor and guide committers and contributors to the project
+- Ensure continued health of project
+  - Adequate test coverage to confidently release
+  - Tests are passing reliably (i.e. not flaky) and are fixed when they fail
+- Ensure a healthy process for discussion and decision making is in place
+- Work with other project owners to maintain the project's overall health and
+  success holistically
```

Comment:
May be we should also develop a section which talks about periodic election of this role, where people could nominate, and have it voted by the community, in such way it would be transparent and free to change.

Reply:
I think an election process would be great, but think we should avoid the guise of a "single-vendor" community voting their favored folks into place, if you get what I mean.

Reply:
Yeah, a robust election process (especially with protections for single-vendor domination) is complex. This was my last effort at creating one: https://wiki.lfnetworking.org/display/ODL/TSC+Election+Process

Reply:
I agree @Oats87 , we need to think that it's a good concern. I was thinking of how people could get rid of me if I did a bad job ':D :-)

Reply:
Let's keep that discussion to a separate PR.